scm/doc/TechGuide/chap_quick.rst

```diff
@@ -129,74 +123,57 @@ System Requirements, Libraries, and Tools
 The source code for the SCM and CCPP components is in the form of
 programs written in FORTRAN 90 (with some required features from the 
 FORTRAN 2008 standard), and C. In addition, the model I/O
-relies on the NetCDF libraries. Beyond the standard scripts, the build
+relies on the NetCDF libraries, as well as the NCEP libraries ``bacio``, ``sp`` and ``w3emc``.
```

Comment:
FYI. @scrasmussen plans to have these dependencies removed from the CCPP before the release.